src/DataFixtures/ORM/LoadAdherentMessageData.php

```diff
@@ -51,6 +51,7 @@ public function getDependencies()
     {
         return [
             LoadAdherentData::class,
```

Comment:
`LoadAdherentData` is no longer required here, because `LoadCommitteeData` depends on `LoadAdherentData`

Reply:
done